swap_meet/item.py

```diff
@@ -1,2 +1,37 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0):
+        if id is  None:
+            self.id =  uuid.uuid4().int 
+        else:
+            self.id = id 
+        
+        if isinstance(condition, (int, float)):
+            if 0 <= condition <= 5:
```

Comment:
Same concern here, not sure if raising an exception is the behavior you really want although in some cases, it might be! It's probably better to design whatever depends on `condition` to handle negative numbers or floats or those greater than 5 accordingly.